### PR TITLE
Fixed the `ffq` tests

### PR DESCRIPTION
**Pull request by muffato**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by SPPearce** [comment]
The md5sums are non-deterministic here, for the multiple sample case.
For SRX7347523, the file details are sometimes the fastq files first, sometimes the sra objects first:
image
Suggest we use contains instead.

File: tests/modules/nf-core/ffq/test.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@
     - ffq
   files:
     - path: output/ffq/SRR9990627.json
-      md5sum: 68db6114ef348f721c139cd4c9534f73
+      md5sum: b69baf0c394bb7e6ea3af7dfc69f5b45
```
Comment (SPPearce) [inline tests/modules/nf-core/ffq/test.yml:7]:
```suggestion
      contains: 
         - SRR9990627
```

File: tests/modules/nf-core/ffq/test.yml [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 
 - name: ffq test_ffq_multiple_ids
   command: nextflow run ./tests/modules/nf-core/ffq -entry test_ffq_multiple_ids -c ./tests/config/nextflow.config  -c ./tests/modules/nf-core/ffq/nextflow.config
   tags:
     - ffq
   files:
     - path: output/ffq/metadata.json
-      md5sum: af253a5587b54520f01a0ff5715b0c0a
+      md5sum: 275fba1ce9476d4eea1814c3260bd63f
```
Comment (SPPearce) [inline tests/modules/nf-core/ffq/test.yml:15]:
```suggestion
      contains: 
         - SRR9990627
         - SRX7347523
```

**Review by SPPearce — COMMENTED** [review]
I'm never great at getting the `contains` syntax correct the first time

**Comment by muffato** [comment]
Let's see if that works 🤞🏼

**Comment by muffato** [comment]
Tests now pass, @SPPearce !

**Pull request merged**